modules/lfs/shared.go

```diff
@@ -14,11 +15,15 @@ import (
 const (
 	// MediaType contains the media type for LFS server requests
 	MediaType = "application/vnd.git-lfs+json"
-	// Some LFS servers offer content with other types, so fallback to '*/*' if application/vnd.git-lfs+json cannot be served
-	AcceptHeader    = "application/vnd.git-lfs+json;q=0.9, */*;q=0.8"
-	UserAgentHeader = "git-lfs"
+	// AcceptHeader Some LFS servers offer content with other types, so fallback to '*/*' if application/vnd.git-lfs+json cannot be served
+	AcceptHeader = "application/vnd.git-lfs+json;q=0.9, */*;q=0.8"
 )
 
+// UserAgentHeader Add User-Agent for gitea's self-implemented lfs client,
+// and the version is consistent with the latest version of git lfs can be avoided incompatibilities.
+// Some lfs servers will check this
+var UserAgentHeader = fmt.Sprintf("git-lfs/3.6.0 (Gitea; %s %s; %s)", runtime.GOOS, runtime.GOARCH, runtime.Version())
```

Comment:
```suggestion
var UserAgentHeader = "git-lfs/3.6.0 (Gitea)"
```

I believe some users won't like to leak their "server information". And we do not need to call the Sprintf now (faster)